pkg/addons/addons_gcpauth.go

```diff
@@ -98,29 +101,28 @@ func enableAddonGCPAuth(cfg *config.ClusterConfig) error {
 	}
 
 	if creds.JSON == nil {
-		out.WarningT("You have authenticated with a service account that does not have an associated JSON. The GCP Auth requires credentials with a JSON file to in order to continue. The image pull secret has been imported.")
+		out.WarningT("You have authenticated with a service account that does not have an associated JSON. The GCP Auth requires credentials with a JSON file in order to continue. The image pull secret has been imported.")
 		return nil
 	}
 
 	// Actually copy the creds over
-	f := assets.NewMemoryAssetTarget(creds.JSON, credentialsPath, "0444")
+	f := assets.NewMemoryAssetTarget(creds.JSON, credentialsPath, readPermission)
 
-	err = r.Copy(f)
-	if err != nil {
+	if err := r.Copy(f); err != nil {
 		return err
 	}
 
 	// First check if the project env var is explicitly set
 	projectEnv := os.Getenv("GOOGLE_CLOUD_PROJECT")
 	if projectEnv != "" {
-		f := assets.NewMemoryAssetTarget([]byte(projectEnv), projectPath, "0444")
+		f := assets.NewMemoryAssetTarget([]byte(projectEnv), projectPath, readPermission)
 		return r.Copy(f)
 	}
 
-	// We're currently assuming gcloud is installed and in the user's path
+	// We're currently assuming gCloud is installed and in the user's path
```

Comment:
The name of the binary is gcloud, not gCloud

Reply:
Thanks for the confirmation - applied!